docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -0,0 +1,161 @@
+  - **26502** for the Zeebe brokers and Zeebe gateway communication
+
+## Limitations
+
+- This is only about a dual-region setup and does not replicate any multi-region setup with more than two Kubernetes clusters
+  - a multi-region setup of 3+ clusters will behave differently due to for example the raft protocol within Zeebe
+- Camunda 8 must be installed with the [Camunda Helm chart](https://github.com/camunda/camunda-platform-helm)
+  - a docker installation or plain JAR installation is not supported
+- Looking at the whole Camunda Platform, it's **active-passive**, while some key components are active-active
+  - meaning there's always one primary and one secondary region for serving active user traffic
+  - serving traffic to both regions will result in a detachment of the WebApps and users seeing different results in Operate and Tasklist
+- The user is responsible for detecting a regional failure and executing the [operational procedure](<-- TODO: link -->)
+- Identity is not supported
+  - Multi-tenancy does not work
+  - Role Based Access Control (RBAC) does not work
+  - the reasoning behind it is that Identity brings its own problems along that require more work on our side
+    - those namely being a multi-region PostgreSQL database and aligning cluster secrets of identity to ensure RBAC works properly across regions
+- Optimize is not supported
+  - this is due to strict coupling with Identity
+- Connectors are not supported
+  - this is due to its coupling with Operate, which has a negative side effect on inbound connectors and can result in race conditions
+- Only a maximum latency of 100 ms between the regions is supported
+- Only specific combinations of Zeebe broker counts and replications factors are supported <!-- TODO: Maybe Deepthi can elaborate on this and how a customer can figure out a setup -->
+- During the fallback procedure, there’s a small chance that some data will be lost for the WebApps
+  - Due to the difference in sequence position tracking for exporters to different Elasticsearch locations <!-- TODO: Maybe for Deepthi to elaborate furthe r -->
+- Zeebe automatic cluster scaling
+  - consider the cluster to be static
+- Web-Modeler is a standalone component and will not be considered further.
+  - Modeling applications can operate independently outside of the automation clusters.
+- Following topics are out of scope and have not yet been explored further and may or may not work well with the dual-region setup
+  - managed OpenSearch usage
+  - OpenShift support
+  - Service meshes
+
+<!-- TODO: Should there be a disclaimer on we're trying to improve the support of Identity, ... with the next version? -->
+
+## Considerations
+
+Multi-region setups in itself bring their own complexity:
+
+- Managing multiple Kubernetes clusters and their deployments across regions
+- Monitoring and alerting
+- Increased costs of multiple clusters and cross-region traffic
+- Data consistency and synchronization challenges
```

Comment:
just a side note since I see a lot of `e.g.` usage, DevEx published a [styling update](https://camunda.slack.com/archives/C026U8GBNSW/p1709922869101609) a while ago to not use latin abbreviations anymore.